_content/canon/naughty-jataka_dhammika.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "A Naughty Jataka"
+translator: dhammika
+external_url: "https://budblooms.org/2020/04/03/a-naughty-jataka/"
+drive_links:
+  - "https://drive.google.com/file/d/1MLeYy-wHNWAwWBDFnJUjk67rOyFiObw0/view?usp=sharing"
+course: jataka
+tags:
+  - sex
```

Comment:
No need for the sex tag as it isn't really about sex (I assume everyone reading this website already knows not to put it in a tortoise's mouth)